scripts/sponsor_update_prices.py

```diff
@@ -21,27 +21,38 @@
 https://tinyurl.com/book-sponsorship-procedure
 """
 
-import internetarchive as ia
+import sys
+import json
 import requests
+import internetarchive as ia
 from getpass import getpass
 
+#15331003-receipt.json
 BWB_URL = 'https://www.betterworldbooks.com'
 
 
 if __name__ == "__main__":
-    s = requests.Session()
-    s.post('%s/account/login' % BWB_URL, data={
-        'user': 'openlibrary+sponsorship@archive.org',
-        'password': getpass('password: '),
-    })
-    orders = s.get('%s/services/orders.aspx?op=History' % BWB_URL).json()
-    order_ids = [order['OrderID'] for order in orders['Orders']]
-    # NB: We select the *last
-    order_url = '%s/services/orders.aspx?op=Details&OrderID=%s' % (
-        BWB_URL, order_ids[0])
-    order = s.get(order_url).json()
+    if len(sys.argv) > 1:
+        with open(sys.argv[1]) as fin:
+            order = json.load(fin)
+    else:
+        s = requests.Session()
```

Comment:
Could we use `sess` or `session` instead of `s`?  Folks often assume that `s` is a str and that `i` is an int.  Besides, single-letter variable names that span multiple lines of code look so old school these days.